app/main.development.js

```diff
@@ -390,6 +390,40 @@ const launchDCRWallet = () => {
   return dcrwPID;
 };
 
+const readExesVersion = () => {
+  let spawn = require("child_process").spawnSync;
+  let args = ["--version"];
+  let exes = ["dcrd", "dcrwallet", "dcrctl"];
+  let versions = {
+    decrediton: app.getVersion()
```

Comment:
This is pulling the electron versionl, which explains why it is showing 1.4.15 

Should be showing 1.1.0 instead